internal/protoplugin/protoplugin.go

```diff
@@ -199,7 +200,7 @@ func (m *Message) FQMN() string {
 func (m *Message) GoType(currentPackage string) string {
 	var components []string
 	components = append(components, m.Outers...)
-	components = append(components, m.GetName())
+	components = append(components, gogogen.CamelCase(m.GetName()))
```

Comment:
can you add comment to link to the line where CamelSlice is called in gogoproto?

Reply:
Done